templates/index.html

```diff
@@ -6,29 +6,29 @@
       <div class="row mt-3">
         <div class="col-md-4 mx-auto">
           <div class="form-group">
-            <label class="heading" id="label" for="email">Email *</label>
+            <label class="heading" id="label" for="email">Email *Required</label>
             <div class="input-group margin-bottom-sm">
               <span class="input-group-addon"><i class="fa fa-envelope-o fa-fw"></i></span>
-              <input autofocus="" class="form-control" placeholder="Enter email" name="email" title="Email for sending the ISO"
+              <input autofocus="" class="form-control" placeholder="Enter email" name="email" title="Email for the ISO"
                 required="" type="email" pattern="^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)+$"
                 required /></div>
-            <p id="scroll-to">We send you the link to the ready ISO per email.</p>
+            <p id="scroll-to">We will send the ready ISO file to the e-mail you provide</p>
```

Comment:
Only a download link.

Reply:
What I meant here is I think the English of how you wrote it isn't really understandable. I hope my English is understandable as well but I think it is a good idea to change it because I really didn't understand it

Reply:
@imahumanrcw The issue with how you wrote it is that you make it sound like the actual iso file will be mailed as an attachment in the email. However, what is actually sent is the link to where the user can go to download the iso. Change your message to something that specifies that a link will be sent, not the actual file.